core/src/login.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>Login to Busola</title>
+    <meta name="description" content="" />
+    <meta name="keywords" content="logout" />
+
+    <link rel="stylesheet" href="/assets/luigi-core/luigi.css" />
+    <style type="text/css">
+      .fd-page {
+        height: 100vh;
+      }
+      .fd-page__content {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        height: 100vh;
+      }
+      .fd-has-text-align-center {
+        text-align: center;
+      }
+      .info,
+      .hint {
+        text-align: center;
+        margin: 0;
+      }
+      .info {
+        padding-top: 30px;
+        font-size: 36px;
+      }
+      .hint {
+        font-size: 18px;
+        color: #6f7275;
+        padding-bottom: 30px;
+      }
+      .icon {
+        font-size: 2.5em;
+        color: #0b74de;
+      }
+      i {
+        font-style: normal;
+      }
+      #oidc-form,
+      #error {
+        margin-top: 16px;
+      }
+    </style>
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/js-yaml/4.0.0/js-yaml.js"
+      integrity="sha512-LcHBP6wEG9wiB9l6Ok0oHj/OiepnKTPKSIlIy9I21LPhfNFAQzEJVeEJ2sXYFK8p4V/UR+1/pTHo4Cw1yPK7NA=="
+      crossorigin="anonymous"
+      defer
+    ></script>
+  </head>
+  <body>
+    <div class="fd-ui fd-ui--fundamental">
+      <div class="fd-page">
+        <div class="fd-page__content">
+          <section class="fd-section">
+            <div class="fd-has-text-align-center">
+              <i class="icon sap-icon--account" role="presentation"></i>
+              <div class="info" id="headline">Login to Busola</div>
+              <div class="hint" id="message">
+                Upload your kubeconfig file to login
+              </div>
+            </div>
+            <div class="fd-form-item">
+              <input type="file" id="upload-kubeconfig" />
+            </div>
+            <div
+              id="error"
+              style="display: none"
+              class="fd-object-status fd-object-status--critical"
+            >
+              <i
+                class="fd-object-status__icon sap-icon--message-error"
+                role="presentation"
+              ></i>
+              <span class="fd-object-status__text"
+                >Error reading kubeconfig.</span
```

Comment:
Could we add an error message here?

Reply:
I didn't want to show the specific message from caught error, as it wouldn't give user any more information (like `Cannot read property '0' of undefined`).